### ISSUE TITLE
Writing scripts doesn't give you the completions you'd expect

### ISSUE THREAD

**Issue by ckipp01** [scalameta/metals#5288]
### Describe the bug

When you have the following:

```scala
#!/usr/bin/env -S scala-cli shebang

//> using scala 3.3.0
//> using lib com.lihaoyi::os-lib:0.9.1

```

It's obvious that you're writing a script meaning you don't need a main method, any object, etc. However you won't get completions for things you'd want, as Metals still only thinks you should get completions for valid top level things. For example let's say I wanted to try completions `os.<cursor>` to use os.lib, you won't get any completions for it.

### Expected behavior

I'd expect for Metals to know I'm writing a script and allow me to write pretty much anything at the top level.

### Operating system

macOS

### Editor/Extension

Nvim (nvim-metals)

### Version of Metals

0.11.12+77-cce28f19-SNAPSHOT

**Comment by tgodzik** [comment]
Thanks for reporting! I seem to be getting all the completions that I would expect. I thought that in your case this might have something to do with some recent bugs, but it seems to work for 1.0.0-RC2, so this is probably not related.

Are you able to reproduce it every time? How do you start Scala CLI and which version?

![Peek 2023-05-29 23-31](https://github.com/scalameta/metals/assets/3807253/4975f63c-0517-4407-be99-350cb50166c9)

**Comment by ckipp01** [comment]
Huh, odd, yea it happens for me wherever I try it. I'm using 1.0.0 and basically I create the file, run `scala-cli setup-ide project.scala` and then open it. Funny enough it even shows me like the top level completions sometimes _after_ the `.`. Like this:

<img width="422" alt="Screenshot 2023-05-30 at 09 01 25" src="https://github.com/scalameta/metals/assets/13974112/f9febae3-3c0e-45be-9867-e17da2601cc3">

Maybe this is some odd interaction with how nvim is sending in completions here or something. I'll try to dig further and get more info.

**Comment by tgodzik** [comment]
Looks like this broke between 1.0.0-RC2 and 1.0.0, it's most likely related to https://github.com/VirtusLab/scala-cli/issues/2158

We'll need a 1.0.1 release :(

**Comment by tgodzik** [comment]
This is fixed in 1.0.1 :tada: I need to still update to newest one in Metals, but it seems some other issues are present
